pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -145,7 +155,10 @@ stdenv.mkDerivation rec {
     "--cross-prefix=${stdenv.cc.targetPrefix}"
     "--cpu=${stdenv.hostPlatform.uname.processor}"
     (lib.enableFeature guestAgentSupport "guest-agent")
-  ] ++ lib.optional numaSupport "--enable-numa"
+  ] ++ lib.optional enableDocs "--enable-docs"
```

Comment:
Let's use `lib.enableFeature` and friends for new options, like it done for `guestAgentSupport` above.